incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -1670,9 +1677,35 @@ else if (length > MAXIMUM_CLIENT_ID_LENGTH)
                 serverDefinedKeepAlive = keepAlive != connect.keepAlive();
                 keepAliveTimeout = Math.round(TimeUnit.SECONDS.toMillis(keepAlive) * 1.5);
                 doSignalKeepAliveTimeout();
+
+                long exchangeAuth = authorization;
```

Comment:
The term `exchangeAuth` is an `http` term for the `http` `exchange`, i.e. the `http` `request-response`.
Let's rename this to `sessionAuth` or similar to refer to the `mqtt` concept instead.